README.md

```diff
@@ -10,8 +10,8 @@
 ## 📖 Introduction
 
 gorabbit is a RabbitMQ client for Golang. It is a wrapper around [rabbitmq/amqp091-go](github.com/rabbitmq/amqp091-go) library. 
-It provides a simple interface to interact with RabbitMQ. Also, it provides a simple way to create a consumer and a publisher that we call them `jobs`.
-It's good to mentioned that gorabbit handles the reconnection and reconsuming the jobs automatically.
```

Comment:
the orginal text is correct. gorabit handle reconsuming that very important feature.